src/files/XmlFile.ts

```diff
@@ -70,6 +70,11 @@ export class XmlFile {
 
     public commentFlags = [] as CommentFlag[];
 
+    /**
+     * should this file be written to disk
+     */
```

Comment:
```suggestion
    /**
     * Will this file result in only comment or whitespace output? If so, it can be excluded from the output if that bsconfig setting is enabled.
     */
```